pkg/sass-parser/lib/src/expression/selector.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright 2025 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import * as postcss from 'postcss';
+
+import {LazySource} from '../lazy-source';
+import {NodeProps} from '../node';
+import type * as sassInternal from '../sass-internal';
+import * as utils from '../utils';
+import {Expression} from '.';
+
+/**
+ * The initializer properties for {@link SelectorExpression}.
+ *
+ * Unlike other expression types, this can't be initialized by properties alone,
+ * since it doesn't have any properties to set.
+ *
+ * @category Expression
+ */
+export interface SelectorExpressionProps extends NodeProps {
+  raws?: SelectorExpressionRaws;
+}
+
+/**
+ * Raws indicating how to precisely serialize a {@link SelectorExpression}.
+ *
+ * @category Expression
+ */
+// eslint-disable-next-line @typescript-eslint/no-empty-interface -- No raws for a selector expression yet.
+export interface SelectorExpressionRaws {}
+
+/**
+ * An expression representing the current selector in Sass.
```

Comment:
Nit: to match the Dart library `sass_api` docs, mention this is specifically `&`